docs/CHANGELOG.asciidoc

```diff
@@ -35,6 +36,289 @@ This section summarizes the changes in each release.
 * <<release-notes-7.0.0-alpha1>>
 
 --
+
+[[release-notes-7.7.0]]
+== {kib} 7.7.0
+
+See <<breaking-changes-7.7,breaking changes in 7.7>>.
+
+[float]
+[[enhancement-7.7.0]]
+=== Enhancements
+APM::
+* Updates monospace font family variable {pull}57555[#57555]
+* Shows missing permissions message to the user on the Services overview {pull}56374[#56374]
+* Settings list page for managing custom actions {pull}56853[#56853]
+* Divides &quot;Actions menu&quot; into sections to improve readability {pull}56623[#56623]
+* Creates settings page to manage Custom Links {pull}57788[#57788]
+* Creates custom link from Trace summary {pull}59648[#59648]
+* Writes tests for the Custom Link API {pull}60899[#60899]
+* Threshold alerts  {pull}59566[#59566]
+* Adds additional (java) options {pull}59860[#59860]
+* Sync badge {pull}55113[#55113]
+* Client.ip to metadata for RUM transactions {pull}56546[#56546]
+Canvas::
+* Adds Lens embeddables {pull}57499[#57499]
+* Switches to using EUI SuperDatePicker in time filter element {pull}59249[#59249]
+* Adds Visualize embeddable {pull}60859[#60859]
+* Adds argument to open all links in new tab within markdown element {pull}57017[#57017]
+* Updates esdocs datasource {pull}59512[#59512]
+* Improves expression autocomplete {pull}52035[#52035]
+Dashboard::
+* Dashboard/add panel flow {pull}59918[#59918]
+* Moves create new to the top of SavedObjectFinder {pull}56428[#56428]
+Lens::
```

Comment:
Should we combine with Visualize? From user's point of view, Lens is within Visualize